ui/src/containers/Header/index.tsx

```diff
@@ -52,6 +64,25 @@ const Header: React.FC = observer(() => {
         logo={<Brand src={logo} alt="Tekton Hub Logo" onClick={homePage} />}
         headerTools={headerTools}
       />
+      <Modal
+        variant={ModalVariant.small}
+        title="Search tips:"
+        isOpen={isModalOpen}
+        onClose={() => setIsModalOpen(false)}
+      >
+        <Grid>
+          <TextContent>
+            <TextList>
+              <TextListItem>Press `/` to quick focus on search</TextListItem>
+              <TextListItem>Search resources by it&#39;s name, displayName and tags</TextListItem>
+              <TextListItem>
+                Filter resources by tags with searching tags in a specific format e.g.
+                `tags:tagA,tagB`
```

Comment:
not important but I guess we should move all hardcoded text under single place so that in future if we want to enable i18n then it will become easy

Reply:
probably we can cover all this in a follow-up PR

Reply:
May be we can get this added in the constant or store but works for me in a separate patch as well!